File: 01_Language/01_Functions/js/hexdec.js
function hexdec(hexString) {
    hexString = String(hexString).replace(/[^0-9a-fA-F]/g, "")
    return parseInt(hexString, 16)
}

console.log(hexdec("See"))
console.log(hexdec("ee"))
console.log(hexdec("that"))
console.log(hexdec("a0"))

File: 01_Language/01_Functions/python/hexdec.py
# coding: utf-8

import re


def hexdec(hex_string):
    pattern = re.compile(r'[^0-9A-Fa-f]')
    hex_string = pattern.sub("", hex_string)
    return int(hex_string, 16)


if __name__ == '__main__':
    print(hexdec("See"))
    print(hexdec("ee"))
    print(hexdec("that"))
    print(hexdec("a0"))
